src/Microsoft.AspNet.Mvc.Core/ContentResult.cs

```diff
@@ -7,10 +7,30 @@
 using Microsoft.AspNet.Http;
 using Microsoft.AspNet.Http.Features;
 using Microsoft.AspNet.Mvc.Internal;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
 using Microsoft.Net.Http.Headers;
 
 namespace Microsoft.AspNet.Mvc
 {
+    internal static class ContentResultLoggerExtensions
+    {
+        private static Action<ILogger, string, string, Exception> _resultExecuted;
+
+        static ContentResultLoggerExtensions()
+        {
+            _resultExecuted = LoggerMessage.Define<string, string>(LogLevel.Information, 6,
+                "ContentResult for action {ActionName} executed, had ContentType of {ContentType}");
```

Comment:
Definitely, I'm basically just using semi-sensical placeholders here as I assume we'll want to carefully word them.